spec/gutter-container-component-spec.coffee

```diff
@@ -0,0 +1,142 @@
+Gutter = require '../src/gutter'
+GutterContainerComponent = require '../src/gutter-container-component'
+
+describe "GutterContainerComponent", ->
+  [gutterContainerComponent] = []
+  mockGutterContainer = {}
+
+  buildTestState = (sortedDescriptions) ->
+    mockTestState =
```

Comment:
Same here. I think I'd rather see this integration tested with a real presenter.